tests/cli/test_subcommands.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Callable, ContextManager
+
+import pytest
+
+from conda.common.compat import on_win
+from conda.testing import CondaCLIFixture, PathFactoryFixture, TmpEnvFixture
+
+
+@pytest.fixture
+def environment_yml(path_factory: PathFactoryFixture) -> Path:
+    path = path_factory(name="environment.yml")
+    path.write_text(
+        "name: scratch\n"
+        "channels:\n"
+        "  - defaults\n"
+        "dependencies:\n"
+        "  - ca-certificates=2023\n"
+    )
+    return path
+
+
+def test_clean(conda_cli: CondaCLIFixture):
+    out, err, code = conda_cli("clean", "--all", "--yes")
+    assert out
+    assert not err
+    assert not code
+
+
+def test_create(conda_cli: CondaCLIFixture, path_factory: PathFactoryFixture):
+    out, err, code = conda_cli("create", "--prefix", path_factory(), "--yes")
+    assert out
+    assert not err
+    assert not code
+
+
+def test_compare(
+    conda_cli: CondaCLIFixture, tmp_env: TmpEnvFixture, environment_yml: Path
+):
+    with tmp_env() as prefix:
+        out, err, code = conda_cli("compare", "--prefix", prefix, environment_yml)
+        assert out
+        assert not err
+        assert code
+
+
+def test_config(conda_cli: CondaCLIFixture):
+    out, err, code = conda_cli("config", "--show-sources")
+    assert out
+    assert not err
+    assert not code
+
+
+def test_info(conda_cli: CondaCLIFixture):
+    out, err, code = conda_cli("info")
+    assert out
+    assert not err
+    assert not code
+
+
+def test_init(conda_cli: CondaCLIFixture):
+    out, err, code = conda_cli("init", "--dry-run")
+    assert out
+    assert not err
+    assert not code
+
+
+def test_install(conda_cli: CondaCLIFixture, tmp_env: TmpEnvFixture):
+    with tmp_env() as prefix:
+        out, err, code = conda_cli(
+            "install",
+            *("--prefix", prefix),
+            "ca-certificates",
+            "--yes",
+        )
+        assert out
```

Comment:
`assert out is not None` should be the tested

Reply:
`conda_cli` returns `tuple[str, str, int]`, `out` will always be `not None` so that would be a redundant check

what I'm asserting here is whether anything was printed to `stdout` so an empty string is considered a failure